runtime/amd64.S

```diff
@@ -70,6 +70,7 @@
         .globl name; \
         .type name,@function; \
         .align FUNCTION_ALIGN; \
+        endbr64; \
```

Comment:
That's unlikely to work if you put the instruction before the label...

Reply:
Hmm. Yeah you might be onto something there.

Reply:
Yeah, swapped it around. Some tests failed with my first attempt. Now they pass.